# data[i] = distance to obstacle at an angle alpha
# where alpha = (2 pi radian) * i / len

int32 len
float64[] data

# reliable range of detection
float64 range
